composer.json

```diff
@@ -42,6 +42,7 @@
         "mockery/mockery": "^1.4.4",
         "fakerphp/faker": "^1.9.2",
         "squizlabs/php_codesniffer": "^3.2",
+        "slevomat/coding-standard": "^8.15.0",
```

Comment:
Is this required for the phpcs command to work? If so we should document it in the commit message / release notes.

Reply:
And or maybe include it in the system module's composer.json?

Reply:
Yes, it is required for the extra rules we use in the coding standard. We can't include it in the system module however because you cannot include development requirements from a Composer dependency AFAIK.